tests/test_end_to_end.py

```diff
@@ -312,3 +312,69 @@ def test_comment():
     comments = Client(NOTION_ACCESS_TOKEN).get_comments(block_with_comments_id)
     assert comments[0].rich_text.to_plain_text() == "Test Comment"
     assert comments[1].rich_text.to_plain_text() == "Test Comment 2"
+
+
+def test_render_plugin(tmpdir):
+    def _run_n2y(temp_dir, config, render_config):
```

Comment:
Don't we already have this method in this file? Can we re-use  that instead of creating a new one?

Reply:
this way lets me customize the config file